editor/components/default-block-appender/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import classnames from 'classnames';
+import 'element-closest';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { getDefaultBlockName, createBlock } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import BlockDropZone from '../block-drop-zone';
+import { insertBlock } from '../../actions';
+import { getBlockCount } from '../../selectors';
+
+class DefaultBlockAppender extends Component {
+	constructor( props ) {
+		super( props );
+		this.appendDefaultBlock = this.appendDefaultBlock.bind( this );
+	}
+
+	appendDefaultBlock() {
+		const newBlock = createBlock( getDefaultBlockName() );
```

Comment:
Would it make sense to update `createBlock` definition to simplify this call to:
```js
export function createBlock( name = getDefaultBlockName(), blockAttributes = {} ) { ... }

this.props.onInsertBlock( createBlock() );
```

Reply:
I don't know :)